features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreViewModel.kt

```diff
@@ -841,43 +821,4 @@ class ExploreViewModel @Inject constructor(
     fun logEvent(event: String) {
         analyticsService.logEvent(event, bundleOf())
     }
-
-    private val _isBlockchainSynced = MutableLiveData<Boolean>()
-    val isBlockchainSynced: LiveData<Boolean>
-        get() = _isBlockchainSynced
-
-    private val _isBlockchainSyncFailed = MutableLiveData<Boolean>()
-
-    fun monitorBlockchainState() {
-        blockchainStateProvider.observeState()
-            .filterNotNull()
-            .onEach { state ->
-                updateSyncStatus(state)
-            }
-            .launchIn(viewModelScope)
-    }
-
-    private fun updateSyncStatus(state: BlockchainState) {
-        if (_isBlockchainSyncFailed.value != state.isSynced()) {
-            _isBlockchainSynced.postValue(state.isSynced())
-        }
-
-        _isBlockchainSyncFailed.postValue(state.syncFailed())
-    }
-
-    private val _stakingAPY = MutableLiveData<Double>()
-    val stakingAPY: LiveData<Double>
-        get() = _stakingAPY
-
-    fun getStakingAPY() {
-        viewModelScope.launch(Dispatchers.IO) {
-            _stakingAPY.postValue(0.85 * blockchainStateProvider.getMasternodeAPY())
-        }
-    }
-
-    fun getLastStakingAPY() {
-        viewModelScope.launch(Dispatchers.IO) {
-            _stakingAPY.postValue(0.85 * blockchainStateProvider.getLastMasternodeAPY())
-        }
-    }
```

Comment:
Moved to the `MainViewModel` since `ExploreFragment` is hosted in the `WalletActivity` now.